python/py-python-multipart/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-python-multipart
+version             0.0.5
+revision            0
+
+categories-append   devel
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         A streaming multipart parser for Python
+long_description    {*}${description}
+
+homepage            https://github.com/andrew-d/python-multipart
+
+checksums           rmd160  7794ebe17293e2fd7f4491278257822c353df649 \
+                    sha256  f7bb5f611fc600d15fa47b3974c8aa16e93724513b49b5f95c81e6624c83fa43 \
+                    size    32581
+
+python.versions     37 38 39 310
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-atomicwrites \
+                    port:py${python.version}-attrs \
+                    port:py${python.version}-coverage \
+                    port:py${python.version}-mock \
+                    port:py${python.version}-more-itertools \
+                    port:py${python.version}-pbr \
+                    port:py${python.version}-pluggy \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-yaml
```

Comment:
please take another look at these dependencies. Many of them are test dependencies only and since you don't add a test-phase they are not needed. I admit that upstream doesn't do a great job clearly defining its dependencies and for what they are used; but please investigate this a bit more yourself.

Reply:
Okay, thanks, done.

Reply:
I am still not convinced :) 

If I do `grep import` in the directory `python-multipart-0.0.5/multipart` I get:
```
__init__.py:from __future__ import absolute_import
__init__.py:import sys
__init__.py:from ._version import __version__
__init__.py:from .multipart import (
decoders.py:import base64
decoders.py:import binascii
decoders.py:from .exceptions import Base64Error, DecodeError
decoders.py:        from multipart.decoders import Base64Decoder
exceptions.py:import binascii
exceptions.py:from six import PY3
multipart.py:from __future__ import with_statement, absolute_import, print_function
multipart.py:from six import (
multipart.py:from .decoders import *
multipart.py:from .exceptions import *
multipart.py:    from urlparse import parse_qs
multipart.py:    from urllib.parse import parse_qs
multipart.py:import os
multipart.py:import re
multipart.py:import sys
multipart.py:import shutil
multipart.py:import logging
multipart.py:import tempfile
multipart.py:from io import BytesIO
multipart.py:from numbers import Number
```

to me these are all stand library imports except for `six`, which is not listed as a required dependency but probably should be. 

`yaml` is imported in the test directory, all others are not even mentioned in the repository as far as I can tell. Where did you base this list on given that I asked you to take a closer look? ;)

Reply:
Thanks for taking a closer look—I just removed the ones I knew are associated with testing. Actually searching the code, I don't see **any** of these imports used. I removed all of them.

Reply:
Yes, but as I said above `six` is actually imported in the code but not listed by upstream nor in the Portfile - that should still be fixed.

Reply:
Thanks done.